pkg/cmd/openshift-tests/run-disruption/disruption.go

```diff
@@ -127,7 +127,11 @@ func (opt *RunAPIDisruptionMonitorOptions) Run() error {
 	}()
 
 	<-ctx.Done()
+	opt.writeDataToDisk(recorder)
+	return nil
+}
 
+func (opt *RunAPIDisruptionMonitorOptions) writeDataToDisk(recorder monitorapi.Recorder) error {
```

Comment:
eventually this entire command will be removed.  We can talk about adding flags to the monitor for which invariants to run, but we're going to a single command there.

Reply:
for isntance, running in the cluster using the watch-endpoints-slice is closer to what this logically needs.

Reply:
Also, daemonsets don't drain on evict.  Are you sure that's what you intend?

Reply:
>eventually this entire command will be removed
Sure, I can move this to monitor subcommand later. 

>daemonsets don't drain on evict

It should not matter - during upgrade the monitor would receive SIGINT, dump data on disk and stop. InvariantInClusterDisruption.CollectData can process multiple files from one node

Reply:
Implemented - now the test uses Deployment + anti-affinity